app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,51 @@
+/*
+* This is a manifest file that'll be compiled into application.css, which will include all the files
+* listed below.
+*
+* Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+* or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+*
+* You're free to add application-wide styles to this file and they'll appear at the bottom of the
+* compiled file so the styles you add here take precedence over styles defined in any styles
+* defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+* file per style scope.
+*
+*= require bootstrap_and_overrides
+*= require chosen
+*= require_tree .
+*= require_self
+*/
+
+// Font Awesome
+@import "font-awesome-sprockets";
+@import "font-awesome";
+
+
+// Bootstrap body padding for fixed navbar
+
+.org-container {
+  height: 200px;
+
+  #org-logo {
+    width: 60px;
+    height: 60px;
+  }
+}
+
+#mission-div1, #mission-div2, #mission-div3 {
```

Comment:
I don't know what it is as well. it is not created by us. we just keep it in case someone else needs them